examples/c++/ExampleAdd.cpp

```diff
@@ -54,15 +57,27 @@ int main() {
   block_builder.create<mlir::stablehlo::AddOp>(loc, arguments, attributes)
       .getOperation();
 
-  mlir::Operation* op =
-      block_builder
-          .create<mlir::stablehlo::AddOp>(loc, arg0, arguments, attributes)
-          .getOperation();
+  mlir::Operation* op = block_builder
+                            .create<mlir::stablehlo::AddOp>(
+                                loc, tensorType, arguments, attributes)
+                            .getOperation();
   block_builder.create<mlir::func::ReturnOp>(loc, op->getResult(0));
 
-  // verify the module and dump
+  /** verify and dump the module **/
```

Comment:
Nit: prefer `//` for single-line comments.

Reply:
I have used both  multi-line comments (to express high level actions)  and single line comments (express the steps involved). PTAL if it looks OK.